hw/ip/README.md

```diff
@@ -1,4 +1,4 @@
-# Hardware IP Blocks
+# Hardware IP Blocks (HWIP)
```

Comment:
This file will need to be handle autogen IPs dfferently. In fact, it omits rv_plic and alert_handler, and will need to deal with clk/pwr/rstmgr, flash_ctrl, and otp_ctrl (and others?) since they will be generated from templates.